src/components/dev/LocalizationReport.js

```diff
@@ -96,17 +105,22 @@ class LocalizationReport extends Component {
   
   setFilter = (e, { value }) => { this.setState({ filter: value }); };
  
-  filterResults = (results, filter) => {
-    if (filter === 'all') { 
+  filterResults (results, filter) {
+    if (filter === `all`) { 
       return results;
     }
-    let pass = filter === 'true' ? true : false;
+
+    let pass = true;
+    if (filter !== `true`) {
+      pass = false;
+    }
+
     return results.filter((result) => {
       return result.pass === pass;
     });
   };
 
-  compareLocalizations = (modelLocKey, compareLocKey) => {
+  compareLocalizations (modelLocKey, compareLocKey) {
```

Comment:
:+1: I like replacing bound instance methods with regular instance methods wherever they don't need to be passed as callbacks (or don't use `this`)